klam-ssh/v2/authorizedkeys_command.sh

```diff
@@ -33,6 +33,10 @@ fi
 echo "adding user to docker group"
 gpasswd -a ${USER} docker
 
+echo "adding user to passwd file"
+sed -i "/${USER}/d" /etc/passwd
+echo "${USER}:x:$(id -u ${USER}):$(id -g ${USER}):KLAM USER ${USER}:/home/${USER}:/bin/bash" >> /etc/passwd
```

Comment:
@matthewdfuller  I think so, in order to do a docker ps or docker logs in which is shown in the aqua user docs here: https://docs.aquasec.com/docs/user-access-scenario